src/lib/components/ui/TextField/TextField.jsx

```diff
@@ -65,6 +65,16 @@ export const TextField = (props) => {
     rootVariantClass = styles.rootVariantOutline;
   }
 
+  const inlineStyle = (inputSize) => {
```

Comment:
Test is missing for this feature

Reply:
It's been already included in _all props_ test suite. Is it necessary to test it separately?

Reply:
Then is ok.